### ISSUE TITLE
DLD prevents Stronghhold spawning

### ISSUE THREAD

**Issue by Pulz-TCP** [BlackJar72/DoomlikeDungeons#30]
DLD: 1.13.2
Forge: 14.23.5.2854

As the title says- the presence of this mod seemingly prevents the generation of Strongholds.
Replicable by generating a world with and without the mod present- and using /locate Stronhold to check locations.

**Comment by Pulz-TCP** [comment]
Closing the issue- I am suddenly unable to reproduce the problem and am not sure why.

**Comment by BlackJar72** [comment]
It was probably just dumb luck that on a certain world/seed it just happened to overwrite the stronghold (but the portal should be there) or else a rare vanilla bug.  Anyway, glad you aren't having the issue anymore.

**Comment by Pulz-TCP** [comment]
> It was probably just dumb luck that on a certain world/seed it just happened to overwrite the stronghold (but the portal should be there) or else a rare vanilla bug. Anyway, glad you aren't having the issue anymore.

It was a slight issue in my testing methods that made me falsely identify DLD as the cause. I tested 10+ times using differing seeds with different structure + biome generating mods before I found the cause to be SpongeForge. Loading the same world seed without SpongeForge causes the Stronghold to generate- so it's a matter of trial and error as I go through 200 mods to find what SpongeForge doesn't like.

With that said, DLD was one of a few new mods I added when the issue began- so I'll open a new ticket if I find a link.
